lib/public/Util.php

```diff
@@ -325,9 +325,9 @@ public static function getServerHostName() {
 	 */
 	public static function getDefaultEmailAddress($user_part) {
 		$config = \OC::$server->getConfig();
-		$user_part = $config->getSystemValue('mail_from_address', $user_part);
+		$user_part = $config->getSystemValueString('mail_from_address', $user_part);
```

Comment:
We lack type hinting here, would do cast to be safe

```suggestion
		$user_part = $config->getSystemValueString('mail_from_address', (string)$user_part);
```

the other call should be ok.

Reply:
> or just adding type hinting here as well?

Could break compatibility, although the risk should be fairly small here. For nothing should replace or implement this, and the doc always wanted a string, personally I would accept that, too.